CorsixTH/Lua/app.lua

```diff
@@ -688,6 +688,15 @@ function App:fixConfig()
     if key == "height" and type(value) == "number" and value < 480 then
       self.config[key] = 480
     end
+
+    if (key == "scroll_speed" or key == "shift_scroll_speed") and
+      type(value) == "number" and value > 10 then
+      self.config[key] = 10
+    end
+    if (key == "scroll_speed" or key == "shift_scroll_speed") and
```

Comment:
This check apart from the "value" value is identical to the one above. Can probably be written more concisely. Switching on the "value" value only.